### ISSUE TITLE
Bug: React warning for invalid `fetchPriority` prop on DOM element

### ISSUE THREAD

**Issue by samarth5106** [BDutta18/CITY-STYLE#143]
### 📍 Description
While running the application in development mode, the browser console shows a React warning related to an unrecognized DOM property `fetchPriority`.

React does not recognize `fetchPriority` when passed in camelCase to a DOM element. It suggests using lowercase `fetchpriority` if it is intended as a custom attribute.

This warning appears to originate from the `LazyImage.jsx` component.

---

### 🧪 Steps to Reproduce

1. Clone the repository.
2. Navigate to the project folder:
   ```bash
   cd CITY-STYLE
3. Install dependencies:
4. Start the development server:
5. Open http://localhost:5173 in the browser.
6. Open DevTools → Console.

---

### Expected Behavior

No React warnings should appear in the console.
Only valid DOM attributes should be passed to HTML elements.

### Screenshot

<img width="619" height="354" alt="Image" src="https://github.com/user-attachments/assets/b76a2a2a-1a8f-49d5-b743-83c8e56ec517" />
